Makefile

```diff
@@ -51,8 +51,8 @@ RESULTS_SCHEMA_CHECK = python manage.py graphql_schema --out experimenter/result
 RESULTS_TYPES_GENERATE = python manage.py graphql_schema --out experimenter/results/schema.graphql&&yarn workspace @experimenter/results generate-types
 RUFF_CHECK = ruff check experimenter/ tests/
 RUFF_FIX = ruff check --fix experimenter/ tests/
-RUFF_FORMAT_CHECK = ruff format --check --diff . --exclude node_modules
-RUFF_FORMAT_FIX = ruff format . --exclude node_modules
+RUFF_FORMAT_CHECK = ruff format --check --diff .
+RUFF_FORMAT_FIX = ruff format .
```

Comment:
Without this ruff wants to format the database migration

In pyproject.toml `migrations` and `node_modules` are part of the default exclusions, so we probably don't wan't to be overriding that default here